controllers/numaresourcesoperator_controller.go

```diff
@@ -334,8 +339,40 @@ func (r *NUMAResourcesOperatorReconciler) deleteUnusedDaemonSets(ctx context.Con
 				if err := r.Client.Delete(ctx, &ds); err != nil {
 					klog.ErrorS(err, "error while deleting daemonset", "DaemonSet", ds.Name)
 					errors = append(errors, err)
+				} else {
+					klog.V(3).Infof("DS[%s] deleted", ds.Name)
+				}
+			}
+		}
+	}
+	return errors
+}
+
+func (r *NUMAResourcesOperatorReconciler) deleteUnusedMachineConfigs(ctx context.Context, instance *nropv1alpha1.NUMAResourcesOperator, mcps []*machineconfigv1.MachineConfigPool) []error {
+	klog.V(3).Info("Delete MC start")
+	var errors []error
+	var machineConfigList machineconfigv1.MachineConfigList
+	if err := r.List(ctx, &machineConfigList, &client.ListOptions{Namespace: instance.Namespace}); err != nil {
```

Comment:
MachineConfigs are cluster scoped. 
You don't need to mention a Namespace

Reply:
yeah, I will delete it.